src/Frontend/Testing.hs

```diff
@@ -0,0 +1,19 @@
+{-# OPTIONS_GHC -Werror -Wall #-}
+
+module Frontend.Testing
```

Comment:
I would have appended the contents of this module to the end of `Frontend`.  If you want to have this module, could you move the entire `{A,a}ulaTesting` sub-api here?